examples/with-thirdpartyemailpassword/server/index.js

```diff
@@ -0,0 +1,76 @@
+let express = require("express");
+let supertokens = require("supertokens-node");
+let Session = require("supertokens-node/recipe/session");
+let ThirdPartyEmailPassword = require("supertokens-node/recipe/thirdpartyemailpassword");
+let { Google, Github } = ThirdPartyEmailPassword;
+
+const apiPort = 3001;
+const apiDomain = "http://localhost:" + apiPort
+
+supertokens.init({
+    framework: "express",
+    supertokens: {
+        // try.supertokens.io is for demo purposes. Replace this with the address of your core instance (sign up on supertokens.io), or self host a core.
+        connectionURI: "https://try.supertokens.io",
+        // apiKey: "IF YOU HAVE AN API KEY FOR THE CORE, ADD IT HERE",
+    },
+    appInfo: {
+        // learn more about this on https://supertokens.io/docs/thirdpartyemailpassword/appinfo
+        appName: "Demo App",
+        apiDomain: apiDomain,
+        websiteDomain: "http://localhost:3000",
+    },
+    recipeList: [
+        ThirdPartyEmailPassword.init({
+            providers: [
+                Google({
+                    clientId: "",
+                    clientSecret: ""
+                }),
+                Github({
+                    clientId: "8a9152860ce869b64c44",
+                    clientSecret: "00e841f10f288363cd3786b1b1f538f05cfdbda2",
+                }),
+                ThirdPartyEmailPassword.Apple({
+                    clientId: "4398792-io.supertokens.example.service",
```

Comment:
Add as a comment - "for iOS, the client ID being used is the one provided on the frontend. This is used for android and web".

Reply:
Added a description above the ThirdPartyEmailPassword.Apple provider